src/utilities/request.jl

```diff
@@ -121,6 +121,12 @@ function submit_request(aws::AbstractAWSConfig, request::Request; return_headers
         "EC2ThrottledException",
     ]
 
+    if isa(request.content, IO)
+        request.headers["x-amz-content-sha256"] = "UNSIGNED-PAYLOAD"
+    else
+        request.headers["Content-MD5"] = base64encode(
+            digest(MD_MD5, request.content))
```

Comment:
This repository uses the Blue style guide, which puts closing parentheses on the following line in cases like this
```suggestion
            digest(MD_MD5, request.content)
        )
```
Supposedly automatic formatting suggestions are set up but that appears to not have occurred here.